src/swapd/runtime.rs

```diff
@@ -1474,14 +1518,16 @@ impl Runtime {
                     }),
                 )?;
             }
-
             CtlMsg::PeerdReconnected(service_id) => {
-                // set the reconnected service id, if it is not set yet. This
-                // can happen if this is a maker launched swap after restoration
-                // and the taker reconnects
-                if self.peer_service == ServiceId::Loopback {
-                    self.peer_service = service_id;
-                }
+                // Set the reconnected service id. This can happen if this is a
```

Comment:
Maybe this block comment can be at the arm level as CtlMsg above

Reply:
1d1ec6ec54d8ce2dcd74a0f8e909047e10c03eed